CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
+* Add NVIDIA Blackwell architecture support to the generated makefiles [\#8055](https://github.com/kokkos/kokkos/pull/8055)
+* Add support for building C++20 modules [\#8132](https://github.com/kokkos/kokkos/pull/8132)
+* Set GCC minimum version for cxx 20 [\#8130](https://github.com/kokkos/kokkos/pull/8130)
+* Error out for incompatible RDC and BUILD_SHARED_LIBS [\#8196](https://github.com/kokkos/kokkos/pull/8196)
+
+### Incompatibilities (i.e. breaking changes)
+* Raise IntelLLVM minimum for SYCL to 2024.2.1 [\#7918](https://github.com/kokkos/kokkos/pull/7918)
+* Disallow `Kokkos_ENABLE_DEBUG_DUALVIEW_MODIFY_CHECK=OFF` [\#8021](https://github.com/kokkos/kokkos/pull/8021)
+
+### Deprecations
+* Deprecate KOKKOS_MEMORY_ALIGNMENT, KOKKOS_MEMORY_ALIGNMENT_THRESHOLD [\#7873](https://github.com/kokkos/kokkos/pull/7873)
+* Deprecate Kokkos::MemoryManaged [\#8066](https://github.com/kokkos/kokkos/pull/8066)
+* Deprecate `KOKKOS_NONTEMPORAL_PREFETCH_[LOAD|STORE]` [\#8146](https://github.com/kokkos/kokkos/pull/8146)
+
+### Bug Fixes
+* Get the concurrency from provided execution space instances [\#7870](https://github.com/kokkos/kokkos/pull/7870)
+* Fix simd math functions not compiling when ARCH_NATIVE=ON [\#7912](https://github.com/kokkos/kokkos/pull/7912)
+* HIP and SYCL: fixed a bug where a `MDRangePolicy` of rank 4 or more is incorrectly iterated, leading to some iterations being evaluated more than once for large loops [\#7880](https://github.com/kokkos/kokkos/pull/7880)
+* Fix memory leak in internals of the Serial default execution space [\#8042](https://github.com/kokkos/kokkos/pull/8042)
+* Fix memory leak in the initialization functions [\#8074](https://github.com/kokkos/kokkos/pull/8074)
+* HIP: fix a bug a where `ConstantMemory` launch mechanism sporadically fails due to `hipEventSynchronize` error [\#8094](https://github.com/kokkos/kokkos/pull/8094)
+* Fix fallback simd masked reductions using incorrect identity elements [\#8115](https://github.com/kokkos/kokkos/pull/8115)
+* HPX fix to constrain hpx_thread_buffer size used with TeamPolicy setup [\#8147](https://github.com/kokkos/kokkos/pull/8147)
+* Fix subview view constructor to disallow non-strided view types for strided subviews [\#8210](https://github.com/kokkos/kokkos/pull/8210)
+* Work around a compiler bug with GCC 12.{2,3,4} causing segfaults with the View shared allocation tracker [\#8223](https://github.com/kokkos/kokkos/pull/8223)
+* Fix compiling with compilers that provide an mdspan implementation [\#8234](https://github.com/kokkos/kokkos/pull/8234)
+* Fix launch of intermediate size functors in graph on HIP [\#8188](https://github.com/kokkos/kokkos/pull/8188)
+* Add missing fences for async Random init with unified memory [\#8105](https://github.com/kokkos/kokkos/pull/8105)
+* OpenMPTarget / OpenACC: fix out-of-bounds access in `Random_UniqueIndex` under certain circumstances [\#8077](https://github.com/kokkos/kokkos/pull/8077)
```

Comment:
`Random_UniqueIndex` is in `Impl::`
Can you suggest something less cryptic?

Reply:
We could elaborate the description a bit more to be something like:
```
OpenMPTarget / OpenACC: Fix an out-of-bounds access error in `Kokkos Random` pool initialization when thread count exceeds the maximum concurrency of an execution space.
```